hbase/Dockerfile

```diff
@@ -31,16 +31,10 @@ USER stackable
 WORKDIR /stackable
 
 COPY --chown=stackable:stackable hbase/stackable/patches /stackable/patches
-COPY --chown=stackable:stackable hbase/stackable/jmx /stackable/jmx
+COPY --chown=stackable:stackable hbase/stackable/jmx/config${JMX_EXPORTER} /stackable/jmx
 
 # Cache mounts are owned by root by default
 # We need to explicitly give the uid to use which is hardcoded to "1000" in stackable-base
-# And every cache needs its own id, we can't share them between stages because we might delete the caches
```

Comment:
I think we should keep this in

Reply:
Fixed https://github.com/stackabletech/docker-images/pull/701/commits/0afda96d8ab0f590d0c825e23972bab47181bd77